docker-compose.yml

```diff
@@ -14,11 +14,10 @@ services:
       # This ties host port 8000 to nginx container port 80
       # Inside of the nginx config, it's port 80 it forwarded to Django 8000
       # It is a little confusing.
-      - 127.0.0.1:8000:80
+      - 127.0.0.1:80:80
     volumes:
       - ./config/nginx/conf.d:/etc/nginx/conf.d
       - presqt_static_volume:/usr/src/app/staticfiles
-      - presqt_media_volume:/usr/src/app/mediafiles
 
```

Comment:
I wonder if we might need to keep "media" files as the passthrough area when files are keep during transit from one server to another... 

We can always put it back later if needed.

Reply:
That makes sense. I wasn't aware that's what this folder was for.